src/test/java/com/unzer/payment/business/paymenttypes/CardTest.java

```diff
@@ -216,4 +228,255 @@ public void testCardMailInvalidValue() throws HttpCommunicationException {
             assertEquals("Email has invalid format.", e.getPaymentErrorList().get(0).getMerchantMessage());
         }
     }
+
+    @Test
+    public void testCardRegisterRecurring() throws HttpCommunicationException, MalformedURLException {
+        Card card = new Card(VISA_3DS_ENABLED_CARD_NUMBER, "03/99");
+        card.setCvc("123");
+        card = getUnzer().createPaymentType(card);
+
+        Recurring recurring = getUnzer().recurring(card.getId(), null, null, new URL("https://www.meinShop.de"));
+    }
+
```

Comment:
JUnit tests should include assert() or fail()

see.. https://app.codacy.com/gh/unzerdev/java-sdk/pullRequest?prid=7995522

Reply:
done